src/Microsoft.CSharp/tests/Configurations.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netcoreapp-Unix;
```

Comment:
I see your comment below, but this change will make us loose code coverage, if we remove netstandard completely the tests will not run in netfx and uap anymore. If you need netcoreapp to access Reflection.Emit, then add the netcoreapp configuration and include this test file only when TargetGroup == netcoreapp

Reply:
Okay, will do that tomorrow. How come System.Linq.Expressions gets coverage, since I copied the configuration from it? Is there something I missed? (Question just for my education).

Reply:
```
<BuildConfigurations>
      netcoreapp-Unix;
      netcoreapp-Windows_NT;
      uapaot-Windows_NT;
      uap-Windows_NT;
</BuildConfigurations>
```

They have all configurations listed in there, netcoreapp and uap, however they are not being tested in netfx. Something to consider adding netfx or netstandard to S.Linq.Expressions.